tests/libyui/rspec_tmux_tui.rb

```diff
@@ -0,0 +1,76 @@
+require "shellwords"
+
+class TmuxTui
+  class Error < RuntimeError
+  end
+
+  def self.new_session(*args)
+    new(*args)
+  end
+
+  attr_reader :session_name
+
+  def initialize(shell_command, x: 80, y: 24, detach: true, session_name: nil)
+    @shell_command = shell_command
+    @x = x
+    @y = y
+    @detach = detach
+    @session_name = session_name || new_session_name
+
+    system "tmux", "new-session",
+           "-s", @session_name,
+           "-x", @x.to_s,
+           "-y", @y.to_s,
+           *(@detach ? ["-d"] : [] ),
+           "sh", "-c", "#{@shell_command}; sleep 9999"
```

Comment:
is that long sleep still needed? I think you can do in rspec specific expect name like `expect(@tui).to include_in_pane("Last Event")` and have own specific error handler there if you need to somehow capture it.